apps/ledger-live-mobile/src/components/DeviceNanoAction.tsx

```diff
@@ -9,7 +9,7 @@ export type PropsExceptModelId = {
   width: number;
   error?: Error;
 };
-type Props = PropsExceptModelId & {
+export type Props = PropsExceptModelId & {
   modelId?: "nanoS" | "nanoX" | "blue";
```

Comment:
Feels like this should be a `DeviceModelId`

Reply:
Maybe, but this is legacy code and it might have been written this way for a reason.
It is outside the scope of this PR to fix this kind of things.